lib/Interpreter/ValueExtractionSynthesizer.h

```diff
@@ -90,7 +90,7 @@ namespace cling {
 
     // Find and cache cling::runtime::gCling, setValueNoAlloc,
     // setValueWithAlloc on first request.
-    void FindAndCacheRuntimeDecls();
+    bool FindAndCacheRuntimeDecls(clang::Expr*);
```

Comment:
Please add doc for the `Expr` parameter, especially as it's not obvious without reading the implementation.